network_wrangler/transit/projects/add_route.py

```diff
@@ -89,71 +89,66 @@ def _add_route_to_feed(
     add_routes_df = pd.DataFrame(
         [{k: v for k, v in r.items() if k != "trips"} for r in add_routes]
     )
-    routes_df = pd.concat([feed.routes, add_routes_df], ignore_index=True, sort=False)
+    feed.routes = pd.concat([feed.routes, add_routes_df], ignore_index=True, sort=False)
```

Comment:
why is feed.routes being set here but all the others are set at the end? We don't want to change the `feed` object until the last step.

Reply:
I was just following the previous code, changing routes_df. Because I found the previous logic of updating trips_df, stops_df, etc., is not correct (as in each loop they're overwritten to reflect the latest changes), so I changed them to `feed`.